packages/components/organisms/f-checkout/src/components/TermsAndConditions.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div
+        data-test-id="checkout-terms-and-conditions-component"
+        :class="$style['c-checkoutTermsAndConditions']">
+        <i18n
+            path="checkoutTermsAndConditions">
+            <template #termsAndConditions>
+                <a
+                    :href="$t('termsAndConditionsLinkUrl')"
+                    target="_blank">
+                    <span>{{ $t('termsAndConditionsLinkText') }}</span>
+                </a>
+            </template>
+            <template #privacyPolicy>
+                <a
+                    :href="$t('privacyPolicyLinkUrl')"
+                    target="_blank">
+                    <span>{{ $t('privacyPolicyLinkText') }}</span>
+                </a>
+            </template>
+            <template #cookiePolicy>
+                <a
+                    :href="$t('cookiePolicyLinkUrl')"
+                    target="_blank">
+                    <span>{{ $t('cookiePolicyLinkText') }}</span>
+                </a>
+            </template>
+        </i18n>
+    </div>
+</template>
+
+<style lang="scss" module>
+$checkout-width                           : 460px;
```

Comment:
You could move this variable to `common.scss` so you can share it here and where it's referenced `Checkout.vue` (so that it's only set in one place and they always stay in sync if updated).